data/yaml/page-content/homepage.yaml

```diff
@@ -5,102 +5,106 @@ meta:
+      - title: Statistics
+        content: Learn about the statistics that are available for applications.
+        link: /docs/metadata-stats/stats
+      - title: Push Notifications
+        content: "Learn how to send Push Notifications to users that aren't currently active in an application."
         link: /docs/general/push
-      - title: Realtime analytics
-        content: Use our account dashboards or REST APIs to get access to your account analytics in real time.
-        link: /realtime/statistics
-  - title: Ably Integrations
-    description: Start streaming realtime data to various other queues, services and streams with our [Integration functionality](https://ably.com/integrations).
-    defaultCallToAction: See docs
+  - title: Integrations
+    description: Read more about integrating with Ably.
+    defaultCallToAction: Read docs
     cards:
-      - title: Webhooks
-        content: We can notify your servers over HTTP in real time when devices become present, channels become active, or messages are published.
+      - title: Webhooks and serverless functions
+        content: Learn how to notify HTTP endpoints and invoke serverless functions when messages are published.
         link: /docs/general/webhooks
-      - title: Serverless functions
-        content: Allow your serverless functions, such as Amazon Lambda, to be invoked following channel lifecycle events (such as channel creation), presence events (such as members entering or leaving) or messages being published.
-        link: /docs/general/webhooks
-      - title: Message queues
-        flag: Unique to Ably
-        content: Process, transform and respond to realtime data as it happens. Our message queues allow you to do this the right way from your own worker servers.
-        link: /docs/general/queues
       - title: Firehose
-        content: Stream your realtime data published within the Ably platform directly to another streaming or queueing service such as Amazon Kinesis or Apache Storm.
+        content: Learn how to stream data from Ably to other streaming and queueing services such as Kafka, Pulsar and SQS.
         link: /docs/general/firehose
+      - title: Kafka Connector
+        content: Learn how to send data from one or more Kafka topics into Ably channels.
+        link: /docs/general/kafka-connector
+  - title: Other protocols
+    description: Read more about using Ably without an SDK.
+    defaultCallToAction: Read docs
+    cards:
+      - title: MQTT
+        content: Learn how to use MQTT with Ably.
+        link: /docs/protocols/mqtt
+      - title: Server Sent Events (SSE)
+        content: Learn how to use SSE with Ably.
+        link: /docs/protocols/sse
+  - title: API references
+    description: View Ably's API references.
+    defaultCallToAction: View API reference
+    cards:
+      - title: Realtime SDK
+        content: Connect to Ably, and publish and subscribe to messages in realtime.
```

Comment:
I think this is a good place to connect 'interface' to 'SDK' in the description. wdyt?

Reply:
Good shout - I've done this for REST and realtime.